src/modules/transaction/components/BlockDetailsTransactions/BlockDetailsTransactions.css

```diff
@@ -1,5 +1,9 @@
 .content {
   padding: 0 !important;
+
+  & > header ~ a:first-of-type::before {
```

Comment:
Also here. This is a very specific selector and since it doesn't leave a trail on the HTML, it's hard for the next developer to apply changes without breaking your work.